instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -20,18 +22,26 @@ const (
 	GRPCStatusCodeKey = attribute.Key("rpc.grpc.status_code")
 )
 
+// InterceptorFilter is a predicate used to determine whether a given request in
+// interceptor info should be traced. A InterceptorFilter must return true if
+// the request should be traced.
+//
+// Deprecated: Use stats handlers instead.
+type InterceptorFilter func(*InterceptorInfo) bool
+
 // Filter is a predicate used to determine whether a given request in
 // interceptor info should be traced. A Filter must return true if
 // the request should be traced.
```

Comment:
The comment must be updated.

Reply:
Thanks, how about https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5196/commits/ebabbaaa40c6b4f23b722299bf9bbf5fd9e8aafc ?